src/clims/models/substance.py

```diff
@@ -30,8 +30,6 @@ def __init__(self, *args, **kwargs):
 
     project = FlexibleForeignKey('sentry.Project', null=True)
 
-    location = FlexibleForeignKey('clims.Location', null=True)
```

Comment:
Do I understand this correctly that the interpretation is that one substance can be in multiple containers at the same time? Does that make sense?